src/functions/template-tags/general.php

```diff
@@ -60,14 +63,22 @@ function tribe_get_event_label_singular() {
 		 *
 		 * @param string $label The singular version of the Event label, defaults to "Event" (uppercase)
 		 */
-		return apply_filters( 'tribe_event_label_singular', esc_html__( 'Event', 'the-events-calendar' ) );
+		return esc_html(
```

Comment:
In my opinion, none of those should be escaped on this level. They should always be escaped in the template level, to avoid us possibly double escaping these strings.

I talked to @paulmskim, to make sure we were aligned on this, and my bad if I shared a diff opinion on this. Would love to get other devs opinions on this. @borkweb @sc0ttkclark